### PR TITLE
Milestone 5 Book store 

### PR DESCRIPTION
**Pull request by imhilla**
Hi TSE,
Please review my capstone project for the react and redux section. This is the fifth milestone.
I added styling to the project.
I added a descriptive readme section.
I added a progress bar, and chapters section.
Thank you
Hillary

### CODE REVIEW

File: src/components/categoryFilter.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import user from '../img/user.png';
+
+const categories = ['CATEGORIES', 'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+const booksLink = 'BOOKS';
+
+const renderCategories = categories.map(item => (
+  // eslint-disable-next-line no-return-assign
```
Comment (Forison) [inline src/components/categoryFilter.js]: - Do not disable eslint

File: src/containers/booklist.js [hunk 2]
```diff
@@ -2,52 +2,68 @@ import React from 'react';
...
+        handleFilterChange={handleFilterChange}
+      />
+      <div className="books-container">
+        {filtered.map(book => (
           <Book
             id={book.id}
             key={book.id}
+            author={author}
             handleRemoveBook={handleRemoveBook}
             // eslint-disable-next-line react/jsx-props-no-spreading
```
Comment (Forison) [inline src/containers/booklist.js:36]: - Do not disable eslint

File: src/containers/booksform.js [hunk 3]
```diff
@@ -30,53 +30,42 @@ class BooksForm extends React.Component {
...
+              {/* eslint-disable-next-line no-return-assign */}
+              <input type="text" id="fname" name="title" ref={node => (this.inputNode = node)} className="inputLesson" />
+            </div>
+            <div className="dropdown">
+              <select
+                className="custom-select"
+                id="inputGroupSelect01"
+                onChange={this.handleDrop}
+                ref={this.myRef}
+                // eslint-disable-next-line react/destructuring-assignment
```
Comment (Forison) [inline src/containers/booksform.js]: - Do not disable eslint

File: src/reducers/books.js [hunk 4]
```diff
@@ -1,7 +1,11 @@
 /* eslint-disable react/no-unused-state */
```
Comment (Forison) [inline src/reducers/books.js]: - Do not disable eslint

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Changes Required✖️.
# 🙋‍♂️ Hillary
#### You have done a good job on this one so far, your design looks like the original page. However, there are a few changes we have to make to get this project approved🤗.
> Please not that this is the fifth milestone you still have one more milestone to go.
* [ ] Do not make changes to linter configs, check my comment below👇.
* [ ] File not related should not be push to the repo, but rather put their paths in gitignore so they don't get committed. Kindly remove the `debug.log` from repo
* [ ]  The category field should take the value of the selection not being stack on the "choose category". Currently "Choose category does not change even when a new category is selected".
### That is it, kindly make the changes and submit a re-review👨‍🔧.
#### Happy coding👨‍💻
Addo Forison

**Review by shloch — APPROVED** [review]
## Hello @imhilla , 

Perfect execution, the design clone is very very close.Corrections are made too from previous reviews.
bokstore

You get a thumbs up for this. Nothing much to add to the great job.
We ready for next milestone.
Happy coding

**Pull request merged**